### PR TITLE
Travis CI trusty build script

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

File: .ci/deps.sh [hunk 1]
```diff
@@ -58,6 +58,7 @@ sudo apt-get -y --no-install-recommends install $deps $deps_perl $deps_infer
 sudo sed -i '1s/.*/#!\/usr\/bin\/env python2/' /usr/bin/flawfinder
 
 # Update hlint to latest version (not available in apt)
+mkdir -p ~/.apt-cache
```
Comment (jayvdb) [inline .ci/deps.sh]: this will be moved to `.travis.yml`

**Comment by jayvdb** [comment]
The commits will need to be re-arranged, but first step is https://github.com/coala/coala-bears/pull/1130 needs to be merged.

**Comment by jayvdb** [comment]
unack ab4b3e8

**Comment by jayvdb** [comment]
I've removed the `3.6-dev` target environment, as it is consistently failing, but only on PRs, not the push to my fork.  Strange.  https://github.com/coala/coala-bears/issues/1084#issuecomment-267390984

Anyway, that can be investigated after we've got the Travis builds performing better.

**Comment by jayvdb** [comment]
build log https://travis-ci.org/jayvdb/coala-bears/builds/184314389

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      - ubuntu-toolchain-r-test
+      - hvr-ghc
+      - avsm
+    packages:
+      - clang-3.4
+      - espeak
+      - mono-mcs
+      - chktex
+      - luarocks
+      - verilator
```
Comment (Mixih) [inline .travis.yml]: `verilator` is here twice(L46 and L49).

**Comment by Mixih** [comment]
Seems ok. Travis is quite slow right now. I wonder what is causing it?

**Comment by Mixih** [comment]
ack  	1e6c890

**Comment by Mixih** [comment]
ack 1e6c890

**Comment by jayvdb** [comment]
@rultor merge

**Comment by Mixih** [comment]
ack 6d5cfdd

**Comment by Mixih** [comment]
ack 6d5cfdd

**Comment by Mixih** [comment]
ack 13d2e6d 3df3def

**Comment by jayvdb** [comment]
reack 13d2e6d
reack 3df3def

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**